package.json

```diff
@@ -31,6 +31,8 @@
     "draggabilly": "^2.2.0",
     "jquery": "^3.4.1",
     "packery": "^2.1.2",
-    "progressbar.js": "^1.0.1"
```

Comment:
@spaceplesiosaur was this something you did? I don't see any difference but I was wondering where or how the change was made? Is this something a polyfill changes?

Reply:
@ckoga This is something that was automatically changed when I ran the install - what really happened was that it added a comma to that item since it was no longer the last.  Here is a full screenshot:

<img width="1171" alt="Screen Shot 2019-10-28 at 9 47 59 PM" src="https://user-images.githubusercontent.com/24462031/67736295-bf525780-f9cc-11e9-99b7-4f44ce4dd4b9.png">